### PR TITLE
You can now shove simple animals in cryo

### PR DESCRIPTION
**Pull request by MadmanMartian**
can now buckle animals
can now put animals in cryo
slimes and other large mobs can not be put in cryo
made mining mobs large
made slimes large
made wendigo large

:cl:
* rscadd: Simple animals can be put in cryo
* rscadd: Simple animals can be buckled to chairs
* tweak: Adult slimes are now considered larger than their younglings, rather than being the same size.

### CODE REVIEW

**Comment by Dacendeth** [comment]
Does this buckle change mean Ian can ride the janicart?

**Comment by DamianX** [comment]
I don't get why you'd want to put animals in cryo but maybe I'm missing something

**Comment by spoonylove** [comment]
does this mean we can put bees in cryo? because thats what I hope it means.

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -73,15 +73,15 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
...
+	if(!ismob(O))
 		return
 	if(O.loc == user) //no you can't pull things out of your ass
 		return
 	if(user.incapacitated() || user.lying) //are you cuffed, dying, lying, stunned or other
 		return
 	if(O.anchored || get_dist(user, src) > 1 || get_dist(user, O) > 1 || user.contents.Find(src)) // is the mob anchored, too far away from you, or are you too far away from the source
 		return
-	if(istype(O, /mob/living/simple_animal) || istype(O, /mob/living/silicon)) //animals and robutts dont fit
+	if(istype(O, /mob/living/silicon)) //robutts dont fit
```
Comment (jknpj) [inline code/game/machinery/cryo.dm]: issilicon()

**Pull request merged**